ddtrace/context.py

```diff
@@ -226,29 +217,12 @@ def get(self):
 
                 # Any open spans will remain as `self._trace`
                 # Any finished spans will get returned to be flushed
-                opened_spans = []
-                closed_spans = []
-                for span in trace:
-                    if span._finished:
-                        closed_spans.append(span)
-                    else:
-                        opened_spans.append(span)
-
-                # Update trace spans and stats
-                self._trace = opened_spans
-                self._finished_spans = 0
-
-                return closed_spans, sampled
+                self._trace = [t for t in self._trace if not t._finished]
+
+                return finished_spans, sampled
```

Comment:
Simplifying the left hand side of this 👍 💯

Reply:
Does that mean I get an approval? 👼